api/types/src/transaction.rs

```diff
@@ -1162,13 +1162,19 @@ pub struct WebAuthnSignature {
 impl VerifyInput for WebAuthnSignature {
     fn verify(&self) -> anyhow::Result<()> {
         let public_key_len = self.public_key.inner().len();
+        let signature_len = self.signature.inner().len();
 
         // Currently only takes Secp256r1Ecdsa. If other signature schemes are introduced, modify this to accommodate them
         if public_key_len != PUBLIC_KEY_LENGTH {
             bail!(
                 "The public key provided is an invalid number of bytes, should be {} bytes but found {}. Note WebAuthn signatures only support Secp256r1Ecdsa at this time.",
                 secp256r1_ecdsa::PUBLIC_KEY_LENGTH, public_key_len
             )
+        } else if signature_len > MAX_WEBAUTHN_SIGNATURE_BYTES {
```

Comment:
Btw, I don't think this limit actually matters for the validator code... I think where you wanna enforce a limit is [in here](https://github.com/aptos-labs/aptos-core/blob/f2a61d0b906d1c0c574e25f15be1f6896593c400/types/src/transaction/webauthn.rs#L150).

Reply:
hmm i think you're right, thanks for the callout. Do you think it makes sense to put it in the `try_from`? I think it might be a better idea to put it in the signature's `verify` function instead, but maybe there's something you see that I don't here.

Also if you want to enforce the limit on zkid you may also want to make the same change. Right now i think `MAX_ZK_SIGNATURE_BYTES` is only being checked at `verify_input`

Reply:
That `try_from` isn't actually used anywhere lolz

Reply:
See the updated description above. I worry about setting an arbitrary size limit on a variable sized field, if we needed to change it again in the future it would not be backwards compatible. I want to, at the very least, get this MAX_BYTES check on the signature input in before the 1.10 release. See the description above for more context.